DatadogRUM/Sources/Integrations/TelemetryReceiver.swift

```diff
@@ -166,6 +168,35 @@ internal final class TelemetryReceiver: FeatureMessageReceiver {
         }
     }
 
+    private func send(usage: DatadogInternal.UsageTelemetry) {
+        let date = dateProvider.now
+
+        self.record(event: nil) { context, writer in
```

Comment:
question\blocking/ We're missing any mechanism for volume control, so effectively we will be collecting 20% of all samples (see: [`telemetrySampleRate`](https://github.com/DataDog/dd-sdk-ios/blob/e5a7b3e0e9dc165a5fd85985dd4e623ab042616e/DatadogRUM/Sources/RUMConfiguration.swift#L213-L218)). According to the [original RFC](https://docs.google.com/document/d/1NBe-Qms6mlf0S2009aXmKzUuPR0y5DUZO8K8xMKCR1o/edit?usp=sharing) this is considered too much. We may want to add extra sample rate similar to [`configurationTelemetrySampleRate`](https://github.com/DataDog/dd-sdk-ios/blob/e5a7b3e0e9dc165a5fd85985dd4e623ab042616e/DatadogRUM/Sources/RUMConfiguration.swift#L267-L268).

Reply:
Not touching the sampling priority part of this RP, this will be in a followup when we integrate it.